src/main/java/com/google/api/codegen/transformer/ruby/RubyImportSectionTransformer.java

```diff
@@ -62,6 +63,21 @@ public ImportSectionView generateTestImportSection(GapicInterfaceContext context
     return importSection.build();
   }
 
+  public ImportSectionView generateRakefileAcceptanceTaskImportSection(
+      List<GapicInterfaceContext> contexts) {
+    List<ImportFileView> none = ImmutableList.of();
+    ImportSectionView.Builder importSection = ImportSectionView.newBuilder();
+    importSection.standardImports(none);
+    importSection.externalImports(none);
+    List<ImportFileView> credentialsImports = new ArrayList<>();
```

Comment:
This should be a set since multiple interfaces can map to the same credentials class.

Reply:
Good catch! Fixed, PTAL.